apps/astarte_appengine_api/lib/astarte_appengine_api_web/router.ex

```diff
@@ -23,6 +23,12 @@ defmodule Astarte.AppEngine.APIWeb.Router do
     plug :accepts, ["json"]
     plug LogRealm
     plug Astarte.AppEngine.APIWeb.Plug.AuthorizePath
+    plug Astarte.AppEngine.APIWeb.Plug.LogDeviceAlias
+    plug Astarte.AppEngine.APIWeb.Plug.LogGroupName
+    plug Astarte.AppEngine.APIWeb.Plug.LogDeviceId
+    plug Astarte.AppEngine.APIWeb.Plug.JoinPath
+    plug Astarte.AppEngine.APIWeb.Plug.LogInterface
+    plug Astarte.AppEngine.APIWeb.Plug.LogPath
```

Comment:
Not all controllers using this pipeline need all the plugs. It would be better to factor out only common plugs. For example, the `InterfaceValuesController` had no `LogGroupName` plug, as there is no group involved in retrieving data.